src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/matmul_decompress_convert.cpp

```diff
@@ -278,8 +285,10 @@ std::vector<ov::AnyMap> filter_additional_config_bf16() {
 
 std::vector<CPUSpecificParams> filter_specific_params(bool trySetMlas) {
     std::vector<CPUSpecificParams> specificParams;
-#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+#if defined(OPENVINO_ARCH_ARM)
     specificParams.push_back(CPUSpecificParams{{}, {}, {"acl"}, "acl"});
+#elif defined(OPENVINO_ARCH_ARM64)
+    specificParams.push_back(CPUSpecificParams{{}, {}, {"kleidiai"}, "kleidiai"});
```

Comment:
I would say it should be separate instance for Kleidiai in arm folder, as it done for matmul.

Reply:
Since we are lowering priority of kleidiai impl, `acl` is selected for all SLT tests

Reply:
Is it? {"kleidiai"} arg should force Kleadi impl with higher priority.

Reply:
As was discussed with Egor, the current implementation of priority affects oneDNN primitives only.
Additional feature is required to set priority across all backends via new parameter of runtime info.